pylint_django/tests/input/func_noerror_foreignkeys.py

```diff
@@ -17,8 +21,9 @@ class ISBN(models.Model):
 
 class Book(models.Model):
     book_name = models.CharField(max_length=100)
-    author = models.ForeignKey('Author', on_delete=models.CASCADE)
-    isbn = models.OneToOneField(ISBN, on_delete=models.CASCADE)
+    author = models.ForeignKey(to='Author', on_delete=models.CASCADE)
+    isbn = models.OneToOneField(to=ISBN, on_delete=models.CASCADE)
+    genre = models.ForeignKey(Genre, on_delete=models.CASCADE)
```

Comment:
Maybe add a comment that this last FK is missing the `to` keyword on purpose.  I see there are other models in the test suite which are missing the `to` keyword (and that is fine). I just want to make sure we do not accidentally clean up this use case from the test suite and end up breaking things.